modules/swagger-codegen/src/main/resources/swift3/api.mustache

```diff
@@ -130,7 +130,7 @@ open class {{classname}}: APIBase {
         {{#hasQueryParams}}
         url?.queryItems = APIHelper.mapValuesToQueryItems(values:[
         {{#queryParams}}
-                "{{paramName}}": {{paramName}}{{#hasMore}}, {{/hasMore}}
+                "{{paramName}}": {{paramName}}{{#isEnum}}{{#isContainer}}{{{dataType}}}{{/isContainer}}{{^isContainer}}{{^required}}?{{/required}}.rawValue{{/isContainer}}{{/isEnum}}{{#hasMore}}, {{/hasMore}}
```

Comment:
y - I actually have to make one more fix here - this isn't all the way correct yet.  let me see if I cleanup the readability as well.